core/test/CMakeLists.txt

```diff
@@ -115,14 +126,26 @@ target_include_directories(testRectGrid PUBLIC "${SRC_DIR}" "${CoreModulesDir}"
 target_link_directories(testRectGrid PUBLIC "${netCDF_LIB_DIR}")
 target_link_libraries(testRectGrid LINK_PUBLIC Boost::program_options Catch2::Catch2 "${NSDG_NetCDF_Library}" Eigen3::Eigen)
 
-add_executable(testModelArray
-    "ModelArray_test.cpp"
-    "${CoreSrc}/ModelArray.cpp"
-    "${ModelArrayDetails}/ModelArrayDetails.cpp"
+add_executable(testParaGrid
+    "ParaGrid_test.cpp"
+    "${CoreModulesDir}/ParametricGrid.cpp"
```

Comment:
Issue: Perhaps this change is large enough and this can be done as a follow-up but these test dependencies could probably do with their own cmake variable.

Reply:
A CMake tidy-up would be a good idea after we have the dynamics and physics tied into one model. Since this isn't currently a problem, I suggest making it a future issue.

Reply:
Refactor, that's the software engineery word.

Reply:
I'm not sure a single comment would work when the whole file needs a good hard refactoring. Issue #210 in any case.